encode/av1encode.c

```diff
@@ -806,7 +812,8 @@ static void process_cmdline(int argc, char *argv[])
         else {
             struct stat tmp;
 
-            fstat(fileno(srcyuv_fp), &tmp);
+            int ret = fstat(fileno(srcyuv_fp), &tmp);
+            CHECK_FN(ret, "fstat");
```

Comment:
https://linux.die.net/man/2/fstat: "On success, zero is returned. On error, -1 is returned, and errno is set appropriately." I.e. should be:
```
CHECK_FN(ret < 0, "fstat");
```

And you have same issue in other places below.

Reply:
fstat fseek fseeko will return 0 on success, so let ret == 0 for them